pages/home.html

```diff
@@ -62,15 +66,17 @@
     </a>
     Features
   </h2>
-  <ul>
-    <li>Simulates a 1-on-1 conversation with a tutor.</li>
-    <li>Able to give targeted feedback to learners based on their answers.</li>
-    <li>Allows learners to leave feedback on explorations.</li>
-    <li>Explorations are embeddable in any webpage.</li>
-    <li>Explorations can be played on mobile devices (except for some of the more complex interactions).</li>
-    <li>The exploration editor supports full version history, multiple authors, parameters and analytics dashboards.</li>
-    <li>There are extension frameworks for building new interactions and classification rules.</li>
-  </ul>
+  <p>With Oppia, you can:
+    <ul>
+      <li>simulate a 1-on-1 conversation with a tutor,</li>
+      <li>give targeted feedback to learners based on their answers (tutors),</li>
+      <li>leave feedback on explorations (learners),</li>
+      <li>embed explorations in any webpage.</li>
+      <li>play explorations on mobile devices (except for some of the more complex interactions),</li>
+      <li>have support on full version history, multiple authors, parameters and analytics dashboards in the exploration editor,</li>
+      <li>build new interactions and classification rules using extension framework.</li>
```

Comment:
"Full version history" means that you can see and go back to previous versions. Here's the Wikipedia analogue: [link](https://en.wikipedia.org/w/index.php?title=Florida_Keys&action=history).

Good call on this not being clear; feel free to reword it.